src/test/resources/features/list.feature

```diff
@@ -5,4 +5,5 @@ Feature: List actions
     Given I log in as admin user
     When I select TestBoard board
     And I create a new list with "TestList" as title
```

Comment:
Please justify this line please 2 spaces.

Reply:
Solved